modules/administration-guide/pages/configuring-storage-sizes.adoc

```diff
@@ -57,4 +57,11 @@ spec:
 <2> Specify a claim size on the next line or omit the next line to set the default claim size value. The specified claim size is only used when you select this storage strategy.
 <3> The claim size must be specified as a link:https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/quantity/[{kubernetes} resource quantity]. The available quantity units include: `Ei`, `Pi`, `Ti`, `Gi`,  `Mi` and `Ki`.
 
-pass:[<!-- vale RedHat.CaseSensitiveTerms = YES -->]
\ No newline at end of file
+pass:[<!-- vale RedHat.CaseSensitiveTerms = YES -->]
+
+[IMPORTANT]
+====
+Manually modifying a PVC on the cluster that was provisioned by {prod-short} is not officially supported and may result in unexpected consequences.
+
+If you want to resize a PVC, modify the default workspace storage size in the `CheCluster` custom resource.
```

Comment:
Sorry, I should have mentioned this earlier, but this is also unsupported :melting_face: Changing the default workspace storage size in the Che Cluster custom resource won't result in the existing PVCs being resized.

You'd have to delete and re-create your workspace (or delete the PVC and have it be automatically re-created, but this too is not _officially_ supported and only works for the per-workspace storage strategy as it causes a [bug](https://github.com/devfile/devworkspace-operator/issues/1258) with the per-user storage strategy).

So, it seems like the documentation we're adding here is either only a warning (that modifying the PVC on the cluster is not supported). We could also remark that you have to delete the workspace and re-create it, but that seems redundant, as we've discussed.

Reply:
I see it. So, yes, let's add it as a warning/important note. We definitely need to have this note added to the docs. I've seen users asking for this, and we need to state this even as a warning to make it clear the right/supported procedure for this.

Reply:
Whether in the `device` or `CheCluster` CR, users need to know the proper way to have a workspace PV resized.

Reply:
Since both options that we planned to feature in the note are actually not supported, is there a proper way to resize PVC outside of setting everything up from scratch?

Reply:
Well, in my experiments I noticed that changing the PVC manually and restarting the workspace works. But I'm not sure if the user itself would have permission to do that in its own namespace or if it would have to ask a cluster-admin to perform this change.

Reply:
@AObuchow @rafaeltuelho your call, guys. decide what should be included in the notification and I'll put it in the admin + end user guides.

Reply:
After discussion, we've agreed upon informing the user that manually expanding the PVC is possible, but not officially supported. The user has to restart their workspace after requesting the expansion of their PVC (by modifiying the PVC on the cluster) in order for the PVC to actually get resized (as the workspace pod needs to detach from the PVC to allow expansion to occur).

Reply:
done! I hope I explained it correctly, please have a look.